cluster/saltbase/salt/calico/master.sls

```diff
@@ -0,0 +1,35 @@
+{% if pillar.get('policy_provider', '').lower() == 'calico' %}
+
+calico-etcd:
+  cmd.run:
+    - unless: docker ps | grep calico-etcd
+    - name: >
```

Comment:
gack!  I don't think we want to do this.  Why not run the calico etcd in the same pod as the policy controller?  Ideally we would not need any etcd for calico - would be nice to slave directly off the apiserver.  I know we talked about this a while back, but maybe we should reopen that discussion.